config/environment.rb

```diff
@@ -1,12 +1,5 @@
 # Load the rails application
 require File.expand_path('../application', __FILE__)
 
-#Add timestamps and severity to environment.log
-class Logger
-  def format_message(severity, timestamp, progname, msg)
-    "#{timestamp.strftime('%Y-%m-%d %H:%M:%S')} [#{severity[0]}] #{msg}\n"
-  end
-end
```

Comment:
Can we keep this format for the new logger? Or at least make sure the output is not that verbose (short year, severity short string) as we want to save as many terminal columns as possible for the actual data.